packages/code-du-travail-data/search/indexing/cdtn_documents.py

```diff
@@ -45,6 +44,16 @@ def make_slug(text, seed):
       )
     ), to_lower=True)
 
+# flattend code tree structure to articles
+def flattenCodeArticles(children):
```

Comment:
could we do that in our js script ?

Reply:
yes, we'll lose tree structure though (not important atm)